src/Library/demos/List Model/main.js

```diff
@@ -0,0 +1,86 @@
+import Gio from "gi://Gio";
+import Gdk from "gi://Gdk";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const toggle_list_box = workbench.builder.get_object("toggle_list_box");
+const toggle_flow_box = workbench.builder.get_object("toggle_flow_box");
+const reveal_list_box = workbench.builder.get_object("reveal_list_box");
+const reveal_flow_box = workbench.builder.get_object("reveal_flow_box");
```

Comment:
Much simpler :slightly_smiling_face: 
```suggestion
const stack = workbench.builder.get_object("stack");
```